src/components/dialogs/AddStaffNotationDialog.vue

```diff
@@ -297,7 +297,7 @@ export default class AddStaffNotationDialog extends Mixins(DateMixin) {
     // if this is a court order and file has changed, re-validate notation form
     if (this.isCourtOrder && this.enableValidation) {
       await this.$nextTick() // wait for variables to update
-      this.$refs.notationFormRef.validate()
+      this.$refs.notationFormRef && this.$refs.notationFormRef.validate()
```

Comment:
Do not evaluate (run) the second condition if the first is false. This syntax is in lieu of an if statement.

Reply:
Replaced by conditional chaining.